tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/AspNetCoreTestFixture.cs

```diff
@@ -81,9 +83,9 @@ public async Task TryStartApp(TestHelper helper, bool? enableSecurity = null, st
 
         public void Dispose()
         {
-            if (HttpPort is not 0)
+            if (HttpPort is not 0 && ShutdownPath is not null)
             {
-                var request = WebRequest.CreateHttp($"http://localhost:{HttpPort}/shutdown");
+                var request = WebRequest.CreateHttp($"http://localhost:{HttpPort}{ShutdownPath}");
```

Comment:
I wonder if we should check the response here for 200, to make sure we correctly called shutdown. We would have to throw at this point, as `_currentOutput` will be cleared. I don't _think_ it would increase flakiness 😬 🤞

Reply:
With WebRequest, I think it will throw anyways if the status code is not 2xx, so maybe it's okay as-is? I'll keep this as-is for now but this is a good point worth noting if we have shutdown issues